### PR TITLE
[RadioButton]Change useState to useRef and some minor things

### PR DESCRIPTION
**Pull request by DevYuns**
## Description

Changes

1. useState -> useRef
 - React Native official site recommend using `useRef` instead of `useState` when you try to modifying the animated value.
   You can Check [here](https://reactnative.dev/docs/animated).

2. `useNativeDriver: true`
 - As updated to `expo SDK 38`, React native version was also updated to `0.62`. So we don't need platform based conditional statement anymore.

... and some minor changes

It can be operated in all platforms(ios, android, web).

## Checklist

Before you create this PR confirms that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide](https://github.com/dooboolab/dooboo-ui/blob/master/CONTRIBUTING.md) and followed the process outlined there for submitting PRs.
- [x] I signed the [CLA].
- [x] Run `yarn test` or `yarn test -u` if you need to update snapshot.
- [x] Run `yarn lint`
- [x] I am willing to follow-up on review comments in a timely manner.

### CODE REVIEW

File: main/RadioButton/index.tsx [hunk 1]
```diff
@@ -86,7 +86,7 @@ const InnerCircleAnim = ({
   borderRadius,
   color,
 }: ICircleProps): React.ReactElement => {
-  const [circleAnim] = useState(new Animated.ValueXY());
+  const circleAnim = useRef(new Animated.ValueXY()).current;
```
Comment (hyochan) [inline main/RadioButton/index.tsx:89]: nice!

File: main/RadioButton/index.tsx [hunk 2]
```diff
@@ -75,9 +75,9 @@ const getCircleStyles = (size: number, color: string): ICircleProps => {
   return {
     color,
     size,
-    innerSize: size / 2 - size / 8,
+    innerSize: Math.floor(size - size / 3),
```
Comment (hyochan) [inline main/RadioButton/index.tsx:78]: Could you give an explanation of how this improves component?

Comment (DevYuns) [reply main/RadioButton/index.tsx:78]: It makes that inner Circle has more large ratio than before.  It is designed by Sarah.

Screen Shot 2020-07-09 at 11 56 29

Comment (hyochan) [reply main/RadioButton/index.tsx:78]: image

Thanks for the detail. However, it still looks not identical to the new design.

Comment (hyochan) [reply main/RadioButton/index.tsx:78]: Color of the circle, disabled styled and the font size are bit different.

Comment (hyochan) [reply main/RadioButton/index.tsx:78]: Also, the changes on the design weren't stated in the `PR`. I think this can be enhanced in the future work.

**Review by hyochan — APPROVED** [review]

**Pull request merged**